src/role-helpers.js

```diff
@@ -281,6 +281,29 @@ function computeAriaCurrent(element) {
   )
 }
 
+const elementsSupportingDisabledAttribute = new Set([
```

Comment:
I might be missing something, but this seems to me like something that should be defined in `aria-query` because it can change as part of the spec and having it here is like mixing the ARIA logic with our `role` implementation.
Wdyt?

Reply:
Also, I see we have something like this in [`jest-dom`](https://github.com/testing-library/jest-dom/blob/main/src/to-be-disabled.js#L4) too, so it makes me think that this indeed shouldn't belong in one of our packages.

Reply:
I don't want to put more stuff into `aria-query`. `dom-accesibility-api` makes more sense at this point. Feel free to raise a PR.

Reply:
Done: https://github.com/eps1lon/dom-accessibility-api/pull/919
Please lmk if I can change stuff, I'm not happy with putting the change in the `util` file but I'm not familiar with the codebase, so I'm happy to change it.
Thanks!